packages/mux-player/src/template.ts

```diff
@@ -87,12 +125,10 @@ export const content = (props: MuxTemplateProps) => html`
           ? getSrcFromPlaybackId(props.playbackId, { domain: props.customDomain, token: props.tokens.playback })
           : false
       }"
-      cast-stream-type="${[StreamTypes.LIVE, StreamTypes.LL_LIVE].includes(props.streamType as any) ? 'live' : false}"
+      cast-stream-type="${isLive(props) ? 'live' : false}"
```

Comment:
thought(non-blocker): This looks functionally equivalent, so no worries on removing even though it's out of scope.